docs/index.html

```diff
@@ -792,6 +794,55 @@ <h3 class="i18n" id="trx_color">Custom Colors</h3>
     </p>
     <hr>
 
+    <h2>Searching for Information</h2>
+        <div class="row">
+        </div>
+    
+        <h3>Finding Transactions</h3>
+            <div class="row">
+            </div>
+            <p>
+                The MMEX desktop application provides several utilities to help users find specific transactions.  The "Notes" field of transactions can be searched from the account view. At the bottom of the transaction listing in
+                the account view, input your search query in the text box to the right of the magnifying glass. Click the magnifying
+                glass or press the Enter key to search the transactions in that account. The search is case sensitive.
+            </p>
+            
+            <p>
+                The most powerful tool for finding particular transaction information is the Transaction Reports feature, which is 
+                described in more detail <a class="i18n" href="#section21.1">here</a>.
+            </p>
+
+            <hr>
+
+
+        <h3>Finding Payees</h3>
+            <div class="row">
+            </div>
+            <p>
+                In order to find particular payees, open Payee Management using the menu item: Tools&rarr;Organize Payees.<br>
+
+                In the payee management window, use the search field at the bottom to type your query. The search is not case-sensitive,
+                and as you type the payee list will be filtered with any matches.<br>
+
+            </p>
+
+            <p>
+                If you do not know the exact name or spelling of a payee, you can use the <strong>?</strong> (question mark) wildcard as a substitute for a single
+                character, or the <strong>*</strong> (asterisk) wildcard as a substitute for one or more characters.<br>
+                
+                <ins>Examples:</ins>
```

Comment:
Really <ins>? https://www.w3.org/TR/html5/edits.html#the-ins-element

Reply:
Would using the \<u> tag be more appropriate here for underlining? My course used this page as guidance for formatting HTML text, which provides \<ins> as the example tag for underlining: https://www.w3schools.com/html/html_formatting.asp

Reply:
Do not format text with html, only tag document structure. All formatting can be done using CSS later.
You can use `<h4>` here or simple `<p>`

Reply:
Okay, will do. Almost ready to commit the edits, just waiting to finalize formatting internal href's. Thank you @slodki!